server/services/MQConfig.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+    config: {
+        "vhosts": {
+            "test": {
+                "connection": {
+                    "url": process.env.RABBIT_MQ_URL,
+                    "socketOptions": {
+                        "timeout": 1000
+                    }
+                },
+                "exchanges": ["field-data"],
```

Comment:
You probably need a new exchange named `wallet-service-ex` with a queue binding like the following. The consumer of your publication is web-map-query-service.  "field-data-events" queue are meant for events generated by field-data-service.

```
     "exchanges": [
           "wallet-service-ex"
     ],
      "queues": [
        "token-transfer:events"
      ],
      "bindings": [
        "wallet-service-ex[token.transfer] -> token-transfer: events"
      ],
      "publications": {
        "token-assigned": {
          "exchange": "wallet-service-ex",
          "routingKey": "token.transfer"
        }
      }
```

Reply:
BTW, do we have something like a dashboard/panel to check and manage the queue? RabbitMQ I mean? How can I know what kind of exchanges are existing?

Reply:
Yes we do. I will ping you the url and credentials separately. Also, if the rabbitmq doesn't have an exchange and queues that you have specified in the config, it gets created when app starts running.

Reply:
Thanks, @arunbakt for your helpful configuration. I have added the exchange and queue, now the message has been sent to the queue, 
![Uploading Screen Shot 2021-06-21 at 11.45.00 AM.png…]()